sourcemash/api/feeds.py

```diff
@@ -24,25 +24,28 @@ def output(self, key, feed):
 class getUnreadCount(fields.Raw):
     def output(self, key, feed):
         total_item_count = Item.query.filter_by(feed_id=feed.id).count() 
-        
-        if not current_user.is_authenticated():
-            return total_item_count 
-        
```

Comment:
Not needed anymore because feed_fields_status only pings for this value on login_required requests.